### PR TITLE
Add Option class

### PR DESCRIPTION
**Pull request by eliothedeman**
Super basic implementation. Wasn't sure if it made sense to implement as a native go type or not though.  #674

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```
Comment (st0012) [inline .vscode/tasks.json]: Is it just me or these three line's background are different?

Also, I think we shouldn't put specific code editor's setting in the codebase. Is there any reason for this?

File: lib/option.gb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class Option
+    def initialize(name=nil, value=nil)
```
Comment (st0012) [inline lib/option.gb]: can we make the indentation to 2 spaces?

File: vm/class.go [hunk 3]
```diff
@@ -1204,8 +1204,10 @@ func builtinClassCommonInstanceMethods() []*BuiltinMethodObject {
 							loaders, ok := externalClasses[libName]
 							externalClassLock.Unlock()
 							if !ok {
-								return t.vm.InitErrorObject(errors.InternalError, sourceLine, "Can't require \"%s\"", libName)
-
+								err := t.execGobyLib(libName + ".gb")
```
Comment (st0012) [inline vm/class.go]: nice 👍

File: lib/option.gb (hunk 2 above)
Comment (eliothedeman) [reply lib/option.gb]: Ah sorry!

File: .vscode/tasks.json (hunk 1 above)
Comment (eliothedeman) [reply .vscode/tasks.json]: Missed it. Will ad to the .gitignore

**Comment by SD10** [comment]
This is a really nice addition @eliothedeman @st0012 😍

**Pull request merged**